src/components/DayPickerRangeController.jsx

```diff
@@ -107,6 +107,7 @@ export default class DayPickerRangeController extends React.Component {
     };
 
     this.isTouchDevice = isTouchDevice();
+    this.today = moment();
```

Comment:
Whether something is a touch device won't change, but "today" will - imagine a component created at 11:59PM. Let's not cache this value. Maybe it should be a prop that's a function that defaults to returning `moment` - that way, it can be invoked every render, but can be overridden for tests.